packages/scandipwa/src/component/PasswordChangeForm/PasswordChangeForm.form.js

```diff
@@ -51,7 +51,8 @@ export const customerEmailAndPasswordFields = (range, minimunPasswordCharacter)
             inputType: VALIDATION_INPUT_TYPE.password,
             match: (value) => {
                 const password = document.getElementById('password');
-                return password.value === value;
+                const password_confirmation = document.getElementById('password_confirmation');
+                return password.value === value || password_confirmation.value === '';
```

Comment:
I believe, `value` from arguments and `password_confirmation.value` are the same values?
Thus there is no reason to fetch the field element and check whether the value is empty.

Reply:
I see that `MyAccountCreateAccount.component.js` has the same field.
Please check the current issue on this component.
If the issue appears, please fix it there too.
If it doesn't appear there, please use the solution from that component.